exec/java-exec/src/main/resources/rest/storage/update.ftl

```diff
@@ -126,9 +128,9 @@
     }
 
     function deleteFunction() {
-      if (confirm("Are you sure?")) {
-        $.get("/storage/${model.getName()}/delete", serverMessage);
-      }
+      showConfirmationDialog("Are you sure?", function() {
```

Comment:
Maybe we should improve message, something like: `Are you sure you want to delete ....?`

Reply:
Done.